app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.setdefault(name, self)
 
 
 def create_person_list(people: list) -> list:
```

Comment:
add more detail type annotation for list

Reply:
Done